docker-compose.yaml

```diff
@@ -8,88 +6,77 @@ services:
     volumes:
       - redis_data:/data/redis
     networks:
-      - account-service-service
+      - account-service
 
   frequency:
-    image: frequencychain/instant-seal-node:latest
+    image: dsnp/instant-seal-node-with-deployed-schemas:latest
+    # We need to specify the platform because it's the only image
+    # built by Frequency at the moment, and auto-pull won't work otherwise
     platform: linux/amd64
+    # Uncomment SEALING_MODE and SEALING_INTERVAL if you want to use interval sealing.
+    # Other options you may want to add depending on your test scenario.
+    # environment:
+    #   - SEALING_MODE=interval
+    #   - SEALING_INTERVAL=3
+    #   - CREATE_EMPTY_BLOCKS=true
+    # Uncomment below if you want to let the chain run and keep all of the historical blocks
+    # command: --state-pruning=archive
```

Comment:
Question: Can we also support starting with the offchain indexer here now?
account-service requires the offchain indexer for one of the endpoints.

Reply:
Yes, you can pass any Frequency CLI parameters in the `command`